README.md

```diff
@@ -34,19 +35,27 @@ Lastly, a Dockerfile is provided for installing pre-requisite packages & launchi
 
 Please ensure you meet the [pre-requisites](https://rapids.ai/start.html#prerequisites)
 
-Our container extends the base notebooks container. To build:
+Our container extends the base notebooks container. Unlike our Notebooks repo, which comes with the container, Notebooks Extended is meant to be a constatly updating source of community contributions.  We've made it easy to include Notebooks Extended in your RAPIDS container with 3 easy steps:
+
+Step 1: Download your RAPIDS container
+```bash
+docker pull rapidsai/rapidsai:latest
+```
+
+Step 2: Git Pull Notebooks Extended into the folder of your choice (change "#/folder/of/your/choice" into wherever you desire Notebooks Extended to be). 
+
 ```bash
+mkdir #/folder/of/your/choice
```

Comment:
Step 1 looks good... for steps 2 + 3, maybe a little easier to assume/ignore the folder/of/your choice which makes it more verbose? So:

Step 2: Pull the Notebooks Extended git repo
```
git clone https://github.com/rapidsai/notebooks-extended
```

Step 3: Run the jupyter server in the docker container:
```
cd notebooks-extended
docker run --runtime=nvidia --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 -v /folder/of/your/choice/ -it rapidsai/rapidsai:latest
utils/start-jupyter.sh
```